UT4MasterServer/Models/Requests/WaitTimeEstimate.cs

```diff
@@ -0,0 +1,22 @@
+using System.Text.Json.Serialization;
+
+namespace UT4MasterServer.Models.Requests;
+
+public sealed class WaitTimeEsitmate
+{
+	[JsonPropertyName("ratingType")]
+	public string RatingType { get; set; }
+
+	[JsonPropertyName("averageWaitTimeSecs")]
+	public double WaitTimeSeconds { get; set; }
+
+	[JsonPropertyName("numSamples")]
+	public int SampleCount { get; set; }
+
+	public WaitTimeEsitmate(string ratingType, double seconds, int sampleCount)
```

Comment:
EstimateWaitTime

Reply:
I named it `WaitTimeEstimate` because this is returned from endpoint `.../wait_time/estimate`.

Even if that wasn't the case, for some reason it sounds better to me this way too. I guess i think differently.

Reply:
The endpoint is structured differently, it has **wait_time** as location, and **estimate** as method name.

Convention is that method names are always verbs, not nouns.

Also, you had a typo in Estimate.

Reply:
> Convention is that method names are always verbs, not nouns.

But this is the name of a class and its constructor, not a normal method.

Reply:
I thought it was a method, didn't see it was a constructor. Then it is OK. Just fix the typo.